NitroxClient/MonoBehaviours/Gui/MainMenu/LoadingScreenExtras.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using UnityEngine;
+
+namespace NitroxClient.MonoBehaviours.Gui.MainMenu
+{
+    public class LoadingScreenExtras
```

Comment:
Maybe something more explicit like `LoadingScreenVersionText`  I'm not sure that the generic map will be used for other things.